wodles/aws/aws_s3.py

```diff
@@ -366,21 +366,43 @@ def __init__(self, reparse, access_key, secret_key, profile, iam_role_arn,
         """
         AWS Bucket constructor.
 
-        :param reparse: Whether to parse already parsed logs or not
-        :param access_key: AWS access key id
-        :param secret_key: AWS secret access key
-        :param profile: AWS profile
-        :param iam_role_arn: IAM Role
-        :param bucket: Bucket name to extract logs from
-        :param only_logs_after: Date after which obtain logs.
-        :param skip_on_error: Whether to continue processing logs or stop when an error takes place
-        :param account_alias: Alias of the AWS account where the bucket is.
-        :param prefix: Prefix to filter files in bucket
-        :param suffix: Suffix to filter files in bucket
-        :param delete_file: Whether to delete an already processed file from a bucket or not
-        :param aws_organization_id: The AWS organization ID
-        :param discard_field: Name of the event field to apply the regex value on
-        :param discard_regex: REGEX value to determine whether an event should be skipped
+        Parameters
```

Comment:
This entire block should be placed inside the class docstring above the init function.

Reply:
Fixed in 4397511.